### PR TITLE
fix(discoverv2): ui colors

### PR DESCRIPTION
**Pull request by doralchan**
Screen Shot 2019-12-03 at 4 34 53 PM
Screen Shot 2019-12-03 at 4 34 45 PM

### CODE REVIEW

File: src/sentry/static/sentry/app/components/contextData.jsx [hunk 1]
```diff
@@ -125,7 +123,7 @@ class ContextData extends React.Component {
           <span
             key="value"
             className={
-              (valueInfo.isString ? 'val-string' : 'val-repr') +
+              (valueInfo.isString ? 'val-string' : '') +
```
Comment (davidenwang) [inline src/sentry/static/sentry/app/components/contextData.jsx]: do we no longer need 'val-repr'? I'm not too familiar with contextData, but if it's not needed we could remove it from the less file as it seems to not be used anywhere else.

File: src/sentry/static/sentry/less/group-detail.less [hunk 2]
```diff
@@ -1691,13 +1691,6 @@ span.val {
   margin: -2px -4px;
 }
 
-.val-string-multiline {
```
Comment (davidenwang) [inline src/sentry/static/sentry/less/group-detail.less]: Does this still need to be used for multi line strings if `valueInfo.isMultiLine` is `true`?

Comment (doralchan) [reply src/sentry/static/sentry/less/group-detail.less]: Good catch - I'll remove this

File: src/sentry/static/sentry/app/components/contextData.jsx (hunk 1 above)
Comment (doralchan) [reply src/sentry/static/sentry/app/components/contextData.jsx]: Gladly removing

File: src/sentry/static/sentry/less/group-detail.less (hunk 2 above)
Comment (doralchan) [reply src/sentry/static/sentry/less/group-detail.less]: Note: This was adding an unnecessary background color and margin.

**Comment by doralchan** [comment]
Going with all purple in code blocks

**Review by markstory — APPROVED** [review]

**Pull request merged**